### ISSUE TITLE
Modal styling

### ISSUE THREAD

**Issue by gracejohnsn** [DineIn2018/dinein#20]
Description: This may be because I'm running the app in a browser and not an iPad, but pop-up modals are transparent which makes it sometimes difficult to decipher information/what actions should be performed in the modal. The one I'm specifically referencing is attached as a screenshot and is from the Create User page when selecting a restaurant.
![screenshot 6](https://user-images.githubusercontent.com/37002835/39413908-9c93d640-4bf6-11e8-8b30-a16163b3e1a6.png)

Severity: low - appears to be just a cosmetic issue

**Comment by kassssc** [comment]
Yes, it seems that firefox does not support the backdrop filter css property we are using.

**Comment by monakona** [comment]
The modals look like that in Chrome too

**Comment by kassssc** [comment]
I see... it definitely works on the iPad.
